### PR TITLE
One more instruction and further refinements

### PR DESCRIPTION
**Pull request by lu-zero**
I'm wondering if we couldn't just use a single `target_feature` at module level.

### CODE REVIEW

File: coresimd/powerpc/altivec.rs [hunk 1]
```diff
@@ -394,12 +394,16 @@ mod sealed {
     }
 
     impl VectorMsums<vector_unsigned_int> for vector_unsigned_short {
+        #[inline]
+        #[target_feature(enable = "altivec")]
```
Comment (gnzlbg) [inline coresimd/powerpc/altivec.rs]: the `assert_instr` are missing

Comment (lu-zero) [reply coresimd/powerpc/altivec.rs]: good call!

Actually looks like the portable instructions do not get mapped correctly.

**Comment by lu-zero** [comment]
the CI meanwhile seems having its share of issues:
```
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
```

**Comment by gnzlbg** [comment]
This needs rebasing on top of master.

**Comment by lu-zero** [comment]
Rebased and added another instruction.

**Comment by alexcrichton** [comment]
Thanks!

**Pull request merged**